LeanplumTests/LeanplumTests.m

```diff
@@ -35,43 +35,43 @@ - (void)testConfiguration
 {
     NSString* host = @"test_host";
     NSString* servlet = @"servlet";
-    
```

Comment:
Ist this just text formatting for this file?

Reply:
i think these parts it formatted the text, but the file has other changes as well